### ISSUE TITLE
Bad formatting in some docstrings

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments#867]
(Original issue 1160 created by tshatch on 2015-10-17T14:47:48.654709+00:00)

While building docs:

```
#!text

reading sources... [100%] languages                                                                                     
.../pygments/test/doc/docs/formatters.rst:398: WARNING: Inline emphasis start-string without end-string.
.../pygments/test/doc/docs/lexers.rst:3226: WARNING: Inline emphasis start-string without end-string.
.../test/doc/docs/lexers.rst:3228: WARNING: Inline emphasis start-string without end-string.

```

**Comment by Anteru** [comment]
(Original issue was assigned to birkenfeld)

**Comment by Anteru** [comment]
(Original comment by tshatch on 2015-10-17T14:56:01.920602+00:00)

I don't recall how to get at the generated source easily; can you take a look?

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2015-10-20T17:10:54.902842+00:00)

Closes #1160: fix docstring markup.

→ <<cset 3982887503bb>>
